supplementary_style_guide/master.adoc

```diff
@@ -39,6 +39,9 @@ include::style_guidelines/graphical-interfaces.adoc[leveloffset=+2]
 // Specific documentation types
 // include::style_guidelines/specific-doc-types.adoc[leveloffset=+2]
 
+// Managed cloud services documentation types
+// include::style_guidelines/managed-services.adoc[leveloffset=+2]
```

Comment:
@cbudz If I remember correctly, some of the guidance here for managed services might override our base style guidance elsewhere (for example the conversational style?).

If that's still true, then I think instead of including this as a section under "Style guidelines", maybe it would be better to include it at the same level. And potentially mention about overriding any of the base guidance, if applicable. So the highest level TOC would be:

- Introduction
- Style guidelines
- Managed services guidelines
- Glossary of terms and conventions

wdyt?

Reply:
hey @bergerhoffer - yes it is possible some guidance would override base style items, so thanks for calling that out and suggesting the TOC. I'll update the PR to reflect that change.